transmart-api-server/README.md

```diff
@@ -4,6 +4,16 @@ The TranSMART API server is an application that provides a REST API
 for a TranSMART database. It does not support the `v1` API. User management and authorisation should be provided
 by a separate identity provider. 
 
+## Patient counts threshold configuration
+
+It is possible to configure a threshold value, below which counts are not available for users with `COUNTS_WITH_THRESHOLD` access permission. The threshold will be applied to the result of a query, when ...
```

Comment:
My mistake, I forgot to finish the description after our discussion regarding the threshold understanding. I will update it.